src/query/parser/promql/parse_test.go

```diff
@@ -353,6 +382,14 @@ var binaryParseTests = []struct {
 	{"up and up", functions.FetchType, functions.FetchType, binary.AndType},
 	{"up or up", functions.FetchType, functions.FetchType, binary.OrType},
 	{"up unless up", functions.FetchType, functions.FetchType, binary.UnlessType},
+
+	// Various spacing
+	{"up/ up", functions.FetchType, functions.FetchType, binary.DivType},
+	{"10 -up", scalar.ScalarType, functions.FetchType, binary.MinusType},
+	{"up*10", functions.FetchType, scalar.ScalarType, binary.MultiplyType},
```

Comment:
Add 'up-up' or smth. like that.

Reply:
https://github.com/m3db/m3/pull/2811/commits/876e4caf6035930fa306a736bd839d31ea8cb755